fontforgeexe/fvfontsdlg.c

```diff
@@ -315,8 +318,10 @@ return( true );
 	}
 	if ( fv==NULL )
 	    InterAskFilename(d->fv,last_amount/100.0);
```

Comment:
Seems to me that we should pass `only_compatible` into `InterAskFilename()` and make the same change there as just below. (Or given that it's only called here just merge the `InterAskFilename()` code into this function.)